src/lenten-lentil-curry.md

```diff
@@ -0,0 +1,37 @@
+# Lenten Lentil Curry
+
+Simple green lentil recipe. Suitable for Lent.
```

Comment:
Capitalization in `Suitable for Lent` seems off hear but correct me if I'm wrong.

Reply:
I apologize for the late response. I think it is correct to capitalize `Lent` here, as it is a specific named period.

Reply:
Ok.